.github/workflows/scripts/generate_leaderboard_and_send_to_lark.py

```diff
@@ -0,0 +1,176 @@
+import os
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+
+import matplotlib.pyplot as plt
+import pytz
+import requests
+import seaborn
+from requests_toolbelt import MultipartEncoder
+
+
+@dataclass
+class Contributor:
+    name: str
+    num_commits_this_week: int
+
+
+def generate_user_engagement_leaderboard_image(github_token, output_path):
+    # request to the Github API to get the users who have replied the most in the last 7 days
+    now = datetime.utcnow()
+    start_datetime = now - timedelta(days=7)
+    start_datetime_str = start_datetime.strftime("%Y-%m-%dT%H:%M:%SZ")
+
+    # prepare header
+    headers = {
+        'Authorization': f'Bearer {github_token}',
+        'Accept': 'application/vnd.github+json',
+        'X-GitHub-Api-Version': '2022-11-28'
```

Comment:
Is it ok to just hard code the version here? I don't know about this arguement.

Reply:
Yes, it should be hard-coded as requested by the Github acitons doc.